### ISSUE TITLE
merge PNOC WXS and additional 24 PNOC003 WGS to existing SNV and CNV files and update histologies.tsv

### ISSUE THREAD

**Issue by kgaonkar6** [PediatricOpenTargets/ticket-tracker#174]
#### What data file(s) does this issue pertain to?
cnv-cnvkit.seg.gz
cnv-controlfreec.tsv.gz
snv-consensus-plus-hotspots.maf.tsv.gz
histologies.tsv

#### What release are you using?

after v8?

#### Put your question or report your issue here.
Add the remaining PNOC samples 

For PNOC003 the BS ID list is :
```
1  BS_0AK4F99X
2  BS_2X60Q1ET
3  BS_B4PPG3X5
4  BS_QFH9XM6Q
5  BS_Y96RP1HJ
6  BS_169P1QCA
7  BS_DRVEFVQ5
8  BS_RWJ0571Q
9  BS_Z3AB3KMP
10 BS_38CD519Z
11 BS_B19AWBBW
12 BS_Z64NEPNE
13 BS_VCE73HPW
14 BS_G65EA38C
15 BS_5Z0GA574
16 BS_7G3AFSCR
17 BS_CD7AXQM8
18 BS_4DYW3T2A
19 BS_2NDRETB2
20 BS_KQRAHH6Y
21 BS_X6H86RTA
22 BS_JD05XR6E
23 BS_JDZX545X
24 BS_4PM2QG9C
```

TBD PNOC008 sample list

**Comment by jharenza** [comment]
let's include this for v9 release cc @runjin326

**Comment by jharenza** [comment]
@kgaonkar6 shouldn't there also be RNA-Seq for PNOC dataset2?

**Comment by runjin326** [comment]
@jharenza, I took a look and confirmed that currently none of the PNOC_dataset_2 RNA-Seq samples are in the combined expression file. Attaching a file containing the `Kids_First_Biospecimen_ID` (for RNA-Seq) and `subject_id` for PNOC_dataset_2 - do we want a separate ticket for this? 
[pnoc008_RNA_biospecimenID.txt](https://github.com/PediatricOpenTargets/ticket-tracker/files/7136986/pnoc008_RNA_biospecimenID.txt)

Currently I am not sure whether PNOC008 has histology file for DNA specimens but I previously used the MAF file to match the DNA samples and I am also attaching a file here, @zhangb1. The `Tumor_Sample_Barcode` column is the `Kids_First_Biospecimen_ID` for the DNA I believe. 
[pnoc008_DNA_biospecimenID.txt](https://github.com/PediatricOpenTargets/ticket-tracker/files/7136987/pnoc008_DNA_biospecimenID.txt)

**Comment by zhangb1** [comment]
@jharenza @runjin326 do you have the `PNOC_dataset_2 sample` BS IDs?

**Comment by runjin326** [comment]
@zhangb1, I currently have all the way to 40 - 
[pnoc_data_set_2_DNA_biospecimenID.txt](https://github.com/PediatricOpenTargets/ticket-tracker/files/7179826/pnoc_data_set_2_DNA_biospecimenID.txt)
Previously I used the MAF file for the particular participant and match the ID there (for some reason I cannot open files in Cavatica delivery project so I couldn't find that for 41 and 42)- maybe @komalsrathi has a better way.

**Comment by zhangb1** [comment]
> @zhangb1, I currently have all the way to 40 -
> [pnoc_data_set_2_DNA_biospecimenID.txt](https://github.com/PediatricOpenTargets/ticket-tracker/files/7179826/pnoc_data_set_2_DNA_biospecimenID.txt)
> Previously I used the MAF file for the particular participant and match the ID there (for some reason I cannot open files in Cavatica delivery project so I couldn't find that for 41 and 42)- maybe @komalsrathi has a better way.

Oh Those are from PNOC008 ? if so ,I think I can get those.

**Comment by komalsrathi** [comment]
If `PNOC_dataset_2 == PNOC008`, then you can just get the cohort participant ids from the most up-to-date PNOC008 clinical data manifest (coded as Research ID) from Kids First data tracker (https://kf-api-study-creator.kidsfirstdrc.org/download/study/SD_8Y99QZJJ/file/SF_AQXKSDNG/version/FV_66NVXNR5) and map those to the most up-to-date adapt histologies file to get the BS ids...

**Comment by zhangb1** [comment]
This has been done see the comment in the ticket  https://github.com/PediatricOpenTargets/ticket-tracker/issues/196

**Comment by zhangb1** [comment]
I didn't update histology file, this will be  done by @ewafula  or @runjin326 ?

**Comment by runjin326** [comment]
> This has been done see the comment in the ticket #196

Thanks so much! 
I will give the histologies for additional PNOC to @ewafula for update and he already added ALL WXS in his histologies.

**Comment by ewafula** [comment]
@runjin326, @zhangb1 I had already include addition TARGET samples (mostly ALL and a few for other cancers) shown in the v10 histories PR README. This are the ones that had previously been missing. But I am still waiting on @bmennis to send the final list of normal/tumor that were actually used in the analysis. There might be more samples than what was analyzed in the current update if v10 histologies file.

**Comment by runjin326** [comment]
@ewafula, thanks for updating this! I believed there are a total of 438 pairs that are analyzed now - once you get the lists, could you please update the histologies file to remove samples that are not analyzed (I think originally it says to have 921 samples)? Thanks so much!
